### PR TITLE
831 plots appear the same

### PR DESCRIPTION
**Pull request by Brennan1994**
image

### CODE REVIEW

**Comment by Brennan1994** [comment]
This pull request includes the fix to the specific issue, as well as removal of some unused classes, and a rename of IASResultsVM to ScenarioResultsVM to be more consistent with the Naming scheme in the UI.

**Review by rnugent3 — APPROVED** [review]
Looks great! Thank you. Shall we plan on handling a normal probability axis separately?

File: HEC.FDA.ViewModel/ImpactAreaScenario/Results/SpecificIASResultVM.cs [hunk 1]
```diff
@@ -8,199 +8,197 @@
...
+          {
+            CurrentResultVM = _damageByDamageCategoryVM;
+            break;
+          }
+        case ANNUAL_EXC_PROB:
+          {
+            CurrentResultVM = _performanceAEPVM;
+            break;
+          }
+        case LONG_TERM_RISK:
```
Comment (rnugent3) [inline HEC.FDA.ViewModel/ImpactAreaScenario/Results/SpecificIASResultVM.cs:167]: this should be labeled long-term exceedance probability - not long-term risk

Comment (Brennan1994) [reply HEC.FDA.ViewModel/ImpactAreaScenario/Results/SpecificIASResultVM.cs]: repaired

**Pull request merged**